app/jwt_decode.py

```diff
@@ -0,0 +1,28 @@
+from fastapi import Depends, HTTPException, status
```

Comment:
move all authentication related files to "authentication" package

Reply:
authentication related files moved to authentication directory